tests/cli/init.rs

```diff
@@ -6,21 +9,32 @@ use mdbook::config::Config;
 /// Run `mdbook init` with `--force` to skip the confirmation prompts
 #[test]
 fn base_mdbook_init_can_skip_confirmation_prompts() {
-    let temp = DummyBook::new().build().unwrap();
+    let temp = DummyBook::new().empty().unwrap();
 
-    // doesn't exist before
-    assert!(!temp.path().join("book").exists());
+    // empty folder
+    assert_eq!(temp.path().read_dir().unwrap().count(), 0);
 
     let mut cmd = mdbook_cmd();
     cmd.args(["init", "--force"]).current_dir(temp.path());
     cmd.assert()
         .success()
-        .stdout(predicates::str::contains("\nAll done, no errors...\n"));
+        .stdout(predicates::str::contains("\nAll done, no errors...\n"))
+        .stderr(predicates::str::contains(
+            "Creating a new book with stub content",
+        ));
 
     let config = Config::from_disk(temp.path().join("book.toml")).unwrap();
     assert_eq!(config.book.title, None);
+    assert_eq!(config.book.language, Some(String::from("en")));
+    assert_eq!(config.book.multilingual, false);
+    assert_eq!(config.book.src, PathBuf::from("src"));
 
     assert!(!temp.path().join(".gitignore").exists());
+    let summary = read_to_string(temp.path().join("src").join("SUMMARY.md")).unwrap();
+    assert_eq!(summary, "# Summary\n\n- [Chapter 1](./chapter_1.md)\n");
+    let chapter_1 = read_to_string(temp.path().join("src").join("chapter_1.md")).unwrap();
+    assert_eq!(chapter_1, "# Chapter 1\n");
+    assert!(temp.path().join("book").exists());
```

Comment:
Since this test is mostly about skipping prompts (and verifying the result of those skipped prompts), perhaps these should be moved to `tests/init.rs`?

Reply:
I can move it there, but this is still a cli-based test so shouldn't it stay in the cli folder?